tests/test_timerit.py

```diff
@@ -1,8 +1,9 @@
 from xdoctest.utils import CaptureStdout
 from timerit import Timer, Timerit
 from functools import partial
-import random
 
+import timerit
+import random
```

Comment:
unused.

Reply:
The `timerit` import is used in `test_timerit_verbose_via_package()`.

Reply:
I didn't add the `import random` line, and in any case it seems to be used in `HackedTime.__init__()`.  The `run_linter.sh` script doesn't report any errors when I run it locally.

Reply:
My bad, not sure why I didn't see it.